src/interpreter/terminator.rs

```diff
@@ -566,14 +575,14 @@ impl<'a, 'tcx> EvalContext<'a, 'tcx> {
 }
 
 #[derive(Debug)]
-struct ImplMethod<'tcx> {
-    method: Rc<ty::Method<'tcx>>,
-    substs: &'tcx Substs<'tcx>,
-    is_provided: bool,
+pub (super) struct ImplMethod<'tcx> {
+    pub (super) method: Rc<ty::Method<'tcx>>,
+    pub (super) substs: &'tcx Substs<'tcx>,
+    pub (super) is_provided: bool,
 }
 
 /// Locates the applicable definition of a method, given its name.
-fn get_impl_method<'a, 'tcx>(
+pub (super) fn get_impl_method<'a, 'tcx>(
```

Comment:
What's with the space between `pub` and `(` everywhere? AFAIK it's non-canonical.